_harp/programme/_overview.md

```diff
@@ -1,38 +1,32 @@
+We believe that high quality education should be accessible to everyone. Our students continue to be involved with our programme long after they graduate. With their support and the work of the broader Founders & Coders community, we continue to offer our programmes for free at the point of delivery.
 
-As of February 2017, we offer two programme locations: [London](/programme/course-information/london), UK and [Nazareth](/programme/course-information/nazareth), Israel.
+Founders & Coders welcomes applicants over 18 from all academic and professional backgrounds. We find that successful students are self-motivated, prepared to work long hours, and committed to collaborative working. Coding also requires a deep curiosity about how things work and a certain stubbornness in the face of repeated failure.
 
-### Pre-course introductory study
+### Self-learning and meetups
 
-Prospective students use online materials to teach themselves the fundamentals of programming and have the option to attend  weekly in-person meetups at the Founders & Coders campus while they are preparing and submitting their applications to the course.
+Prospective students use online materials to teach themselves the fundamentals of programming and have the option to attend weekly in-person meetups at the Founders & Coders campus while they are preparing and submitting their applications to the course.
 
-Visit [our Meetup site](http://www.meetup.com/founderscoders/) to learn more.
+### Prep-course
 
-### 16-week full-time coding bootcamp
+After conducting interviews, we select a group of finalists for an in-person, weekly prep-course, where we provide support completing [our course prerequisites](../apply/prerequisites).
 
-Our programmes in Nazareth and London run three times a year. The dates for our 2017 cohorts are as follows:
- + 20th February - 9th June
- + **26th June - 13th October** ([applications open](http://www.foundersandcoders.com/apply/))
- + 23rd October - 23rd February
+### 16-week full-time coding bootcamp
 
-Our immersive course is peer-led and project-based with sixteen students working in four teams of four. Students take turns delivering seminars, running code reviews and managing projects. The course covers test-driven development, using a full JavaScript stack (JavaScript and Node.js) with both relational and non-relational databases, as well as aspects of UX design and product management.
+Our immersive course is peer-led and project-based with sixteen students working in four teams of four. Students take turns delivering workshops, running code reviews and managing projects. The course covers test-driven development, using a full JavaScript stack (JavaScript and Node.js) with both relational databases, as well as aspects of UX design and product management.
```

Comment:
Also, maybe we should change product management to project management cos I think that's kind of where the focus is...?